packages/design-system-react-native/src/primitives/ButtonAnimated/README.md

```diff
@@ -0,0 +1,59 @@
+# ButtonAnimated
+
+The `ButtonAnimated` component is a React Native button with scale animation for press interactions. It utilizes `react-native-reanimated` for animations.
+
+## Props
+
+### `onPressIn`
+
+Callback function triggered when the button is pressed in.
+
+| **Type**   | **Required** | **Default** |
+| ---------- | ------------ | ----------- |
+| `function` | No           | `undefined` |
+
+### `onPressOut`
+
+Callback function triggered when the button is released.
+
+| **Type**   | **Required** | **Default** |
+| ---------- | ------------ | ----------- |
+| `function` | No           | `undefined` |
+
+### `disabled`
+
+Disables button interactions when set to `true`.
+
+| **Type**  | **Required** | **Default** |
+| --------- | ------------ | ----------- |
+| `boolean` | No           | `false`     |
+
+### `...props`
+
+Additional props passed to the underlying `Pressable` component.
+
+## Usage
+
+```tsx
+import React from 'react';
+import ButtonAnimated from './ButtonAnimated';
```

Comment:
non-blocking: not sure if we are exposing these primitive components. But we may want to change this import statement to be from the perspective of importing them into a component e.g. 
```suggestion
import ButtonAnimated from '../../primitives';
```

Reply:
How would that make a difference?